examples_gbs/run_tutorial_dense.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Dense Subgraph Tutorial
+=========================
+Graphs can be used to model a wide variety of objects: social networks, financial markets,

```

Comment:
```suggestion
Graphs can be used to model a wide variety of concepts: social networks, financial markets,
```
Wasn't sure if a social network was an object 🤔

Reply:
Not sure if a social network is a concept :thinking: 
^joking btw